theme/style.css

```diff
@@ -16,3 +16,22 @@
 }
 
 /* style your grid sections here!  */
+.Gucc h1 {
+  font-family: fantasy;
+  font-size: 36px;
+  font: bolder;
+}
+
+.Gucc h4 {
+  font-size: 36px;
+  font-family: fantasy;
+}
+
+.Gucc ul{
+  font-family: fantasy;
+  align-content: center;
+}
+.Gucc img{
+  align-items: center;
```

Comment:
This rule only works on containers of `display: flex`, which isn't the case here (images are `display: inline` by default)